src/main/java/ruslan/shastkiv/bookstore/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package ruslan.shastkiv.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+
+@Entity
+@Table(name = "users")
+@Data
+@NoArgsConstructor
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NonNull
```

Comment:
Let's be consistent
```suggestion
    @Column(nullable = false)
```
fix for other fields